src/components/Patient/PatientHome.tsx

```diff
@@ -284,173 +283,158 @@ export const PatientHome = (props: {
-                      text={t("vaccinated")}
-                    />
+                      </div>
+                    </div>
                   )}
-                  {patientData.allow_transfer ? (
-                    <Chip
-                      variant="warning"
-                      size="small"
-                      startIcon="l-unlock"
-                      text={t("transfer_allowed")}
-                    />
-                  ) : (
-                    <Chip
-                      startIcon="l-lock"
-                      size="small"
-                      text={t("transfer_blocked")}
-                    />
+                  {patientData?.last_consultation?.assigned_to_object && (
+                    <div>
+                      <p className="text-xs font-normal leading-tight text-gray-600">
+                        {t("assigned_doctor")}:
+                      </p>
+                      <div className="mt-1 flex space-x-2 text-sm font-semibold leading-tight text-gray-900">
+                        <p>
+                          {formatName(
+                            patientData.last_consultation.assigned_to_object,
+                          )}
+                        </p>
+                        {patientData?.last_consultation?.assigned_to_object
+                          .alt_phone_number && (
+                          <a
+                            href={`https://wa.me/${patientData.last_consultation.assigned_to_object.alt_phone_number.replace(/\D+/g, "")}`}
+                            target="_blank"
+                            rel="noreferrer"
+                            className="flex items-center space-x-1 text-xs text-green-500"
+                          >
+                            <CareIcon icon="l-whatsapp" />{" "}
+                            <span>{t("video_call")}</span>
+                          </a>
+                        )}
+                      </div>
+                    </div>
                   )}
 
-                  {patientData.gender === 2 && (
-                    <>
-                      {patientData.is_antenatal &&
-                        isAntenatal(
-                          patientData.last_menstruation_start_date,
-                        ) && (
+                  {patientData.assigned_to_object && (
+                    <div>
+                      <p className="text-xs font-normal leading-tight text-gray-600">
+                        {t("assigned_volunteer")}:
+                      </p>
+                      <p className="mt-1 text-sm font-semibold leading-tight text-gray-900">
+                        {formatName(patientData.assigned_to_object)}
+                      </p>
+                    </div>
+                  )}
+                </div>
+
+                <div>
+                  <div className="ml-auto mt-4  flex flex-wrap gap-3">
+                    {!patientData?.last_consultation && (
+                      <Chip
```

Comment:
Let's switch these out for Shadcn's Badge component (ui/). Otherwise, lgtm.

Reply:
I need to customize the Badge component, right? The existing Badge component does not support additional variants or features like start and end icons, as the Chip component does.

Reply:
Add variants 👍

Reply:
@Jacobjeevan done the changes